mecanumDriveFieldCentric.java

```diff
@@ -142,13 +153,35 @@ public void runOpMode() throws InterruptedException {
 				);
 			//!SECTION - End Base
 
-			//SECTION - Arms
+			//SECTION - Arm
 				//NOTE - Set the power of the arm motors
 				MShoulderLeft.setPower(-leftStickYGP2 * SHOULDER_SPEED);
 				MShoulderRight.setPower(leftStickYGP2 * SHOULDER_SPEED);
 				MElbowLeft.setPower(rightStickYGP2 * ELBOW_SPEED);
 				MElbowRight.setPower(-rightStickYGP2 * ELBOW_SPEED);
-			//!SECTION - End Arms
+			//!SECTION - End Arm
+
+			//SECTION - Claws
+			if (gamepad2.left_trigger > 0) {
+				clawLeft.setPower(gamepad2.left_trigger * CLAW_CLOSE_POWER);
+				clawLeftPassivePower = CLAW_CLOSE_RESIDUAL_POWER;
+			}
+			else if (gamepad2.left_bumper) {
+				clawLeft.setPower(-CLAW_OPEN_POWER);
+				clawLeftPassivePower = 0;
+			}
+			else clawLeft.setPower(clawLeftPassivePower);
+	
+			if (gamepad2.right_trigger > 0) {
+				clawRight.setPower(-gamepad2.right_trigger * CLAW_CLOSE_POWER);
+				clawRightPassivePower = CLAW_CLOSE_RESIDUAL_POWER;
+			}
+			else if (gamepad2.right_bumper) {
+				clawRight.setPower(CLAW_OPEN_POWER);
+				clawRightPassivePower = 0;
+			}
+			else clawRight.setPower(-clawRightPassivePower);
+			//!SECTION End claws
```

Comment:
Looks good!

there are simpler ways we could write these conditionals, but this is perfectly good!

Reply:
The code could be cleaned up a tad, as a bunch of else-if statements give the code a terrible stench, but it looks good to me.